build/bundler.mts

```diff
@@ -62,12 +62,21 @@ async function* getFiles(topDir: string): AsyncGenerator<string, any, undefined>
 	}
 }
 
+async function ensureDirectoryExists(dir: string) {
+	try {
+		await fs.access(dir)
+	} catch {
+		await fs.mkdir(dir, { recursive: true })
```

Comment:
do we need this to be recursive?

Reply:
in the current state of the build process it's not indeed required so i removed it in [a79243b](https://github.com/DarkFlorist/TheInterceptor/pull/1197/commits/a79243ba097b91367f5a4dfb8fc3b1acd9ee81fe)